Resources/Prototypes/_Harmony/Entities/Objects/Misc/syndie_kudzu.yml

```
@@ -1,41 +1,11 @@
 ﻿- type: entity
   id: SyndieKudzu
-  name: kudzu
+  parent: Kudzu
   suffix: Syndicate Flavor
```

```suggestion
  suffix: Syndicate
```
if you want to be a little more descriptive but its fine and gets the point across so more nitpicky than anything.

---

Do you want me to change it? Im completely ok with doing it.

---

If its not much trouble yeah